apps/portals/elportal/src/pages/StudyDetailsPage.tsx

```diff
@@ -49,63 +50,71 @@ export const studyDetailsPageRoutes: RouteObject[] = [
-                }}
-              />
-            ),
-          },
-          {
-            id: 'Methods',
-            title: 'Methods',
-            element: <MarkdownSynapseFromColumnData columnName={'methods'} />,
-          },
-          {
-            id: 'Related Studies',
-            title: 'Related Studies',
-            element: (
-              <DetailsPageContextConsumer columnName={'relatedStudies'}>
-                {({ value }) => {
-                  if (!value) {
-                    return null
-                  }
+      <>
+        <DetailsPageContent
+          content={[
+            {
+              id: 'Study Description',
+              title: 'Study Description',
+              element: (
+                <MarkdownSynapseFromColumnData
+                  columnName={'studyDescription'}
+                />
+              ),
+            },
+            {
+              id: 'Acknowledgement',
+              title: 'Acknowledgement',
+              element: (
+                <MarkdownSynapseFromColumnData columnName={'ackContext'} />
+              ),
+            },
+            {
+              id: 'acknowledgementFull',
+              element: (
+                <MarkdownSynapseFromColumnData
+                  columnName={'acknowledgment'}
+                  MarkdownCollapseProps={{
+                    textDescription: 'full statement',
+                    showCopyPlainText: true,
+                  }}
+                />
+              ),
+            },
+            {
+              id: 'Methods',
+              title: 'Methods',
+              element: <MarkdownSynapseFromColumnData columnName={'methods'} />,
+            },
+            {
+              id: 'Related Studies',
+              title: 'Related Studies',
+              element: (
+                <DetailsPageContextConsumer columnName={'relatedStudies'}>
+                  {({ value }) => {
+                    if (!value) {
+                      return null
+                    }
 
-                  return (
-                    <CardContainerLogic
-                      sqlOperator={ColumnSingleValueFilterOperator.IN}
-                      sql={studiesSql}
-                      {...studyCardConfiguration}
-                      searchParams={{ id: value }}
-                    />
-                  )
-                }}
-              </DetailsPageContextConsumer>
-            ),
-          },
-        ]}
-      />
+                    return (
+                      <CardContainerLogic
+                        sqlOperator={ColumnSingleValueFilterOperator.IN}
+                        sql={studiesSql}
+                        {...studyCardConfiguration}
+                        searchParams={{ id: value }}
+                      />
+                    )
+                  }}
+                </DetailsPageContextConsumer>
+              ),
+            },
+          ]}
+        />
+        <RedirectToURL
```

Comment:
Added a RedirectToURL that only does something if the search matches (LLFS_Metabolomics)